contributing.md

```diff
@@ -84,10 +84,10 @@ export default withSystemProps(
 Categories of system props are exported from `src/system-props`:
 
 * `COMMON` includes color and spacing (margin and padding) props
-* `TYPOGRAPHY` includes `COMMON` and font family, font weight, and line-height
-* `POSITION` includes `COMMON` and positioning props
-* `FLEX_CONTAINER` includes `COMMON` and flexbox props for containers
-* `FLEX_ITEM` includes `COMMON` and flexbox props for items in a flex container
+* `TYPOGRAPHY` includes font family, font weight, and line-height
```

Comment:
Super minor but should this line say "props" somewhere to be consistent with the other lines?
```suggestion
* `TYPOGRAPHY` includes font family, font weight, and line-height props
```